estate/views/estate_property_views.xml

```diff
@@ -5,4 +5,76 @@
         <field name="res_model">estate.property</field>
         <field name="view_mode">tree,form</field>
     </record>
+
+    <record id="estate_prperty_view_tree" model="ir.ui.view">
+        <field name="name">estate.property.tree</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <tree string="Properties">
+                <field name="name" string="Title"/>
```

Comment:
To not duplicate `string="Title"` everywhere you can add it directly in the field `name = fields.Char(required=True, string="Title")` in `estate_property.py`